modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,181 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
+
+This procedure explains how to configure a cluster to host two control planes, each managing independent service mesh data planes. The configuration involves creating separate `{istio}` resources with unique names in independent {istio} system namespaces. Each `{istio}` resource will use a unique `revision` name, which identifies the control planes, workloads, or namespaces associated with it. These `revision` names, applied using injection or `istio.io/rev` labels determine which control plane will inject the sidecar proxy into the application pods.
```

Comment:
Because Istio does it in the canary docs? https://istio.io/latest/docs/setup/upgrade/canary/

Reply:
Aha! Okay, but in the upstream case, you install a specific revision with `istioctl install --set revision=canary` (so, you need to set a value with that exact name). However, with Sail operator, the user never sets this value (and doesn't even need to know it exists), because it's automatically set by the operator from the `IstioRevision` resource name.

Reply:
Ah ok - makes sense. So in our case, then we shouldn't format revision (ie no back ticks).